src/QEQ/fix_qeq.cpp

```diff
@@ -339,6 +343,12 @@ void FixQEq::setup_pre_force(int vflag)
   if (force->newton_pair == 0)
     error->all(FLERR,"QEQ with 'newton pair off' not supported");
 
+  if (force->pair) {
```

Comment:
Not sure if this check is necessary--the suffix styles should be able to fall back to the non-accelerated styles in most cases.

Reply:
this is another case of removing old changes by copying files instead of merging. fixed.